.github/ISSUE_TEMPLATE/4-hosting-request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: ":cloud: Hosting request"
+labels: hosting-request
+about: I want to host my integration in keptn-sandbox
+---
+
+### Description
+
+<!-- Some context around your integration here -->
+
+### Checklist
+- [ ] Repository transfer: <your-repo-here>
+- [ ] GitHub access permissions
+- [ ] Integration author has followed the [hosting guide](https://github.com/keptn-sandbox/contributing/blob/master/CONTRIBUTING.md) 
+
```

Comment:
```suggestion
Make sure your repository has the following files:

- [ ] README.md
- [ ] LICENSE
- [ ] CODEOWNERS (with my git account)
```

Reply:
We can add these pointers to the checklist.

Reply:
Thank you for the suggestion. Those points are already covered in the [hosting guide](https://github.com/keptn-sandbox/contributing/blob/master/CONTRIBUTING.md)

Reply:
https://github.com/keptn-sandbox/contributing/pull/11/files#diff-21e2e95400a474c56f4d169978f9d93899ec13ff6035a8b38bbbde0dfa81543aR14